components/debounce/browser/debounce_rule.cc

```diff
@@ -212,12 +231,14 @@ bool DebounceRule::Apply(const GURL& original_url,
       action_ != kDebounceRegexPath)
     return false;
   // If URL matches an explicitly excluded pattern, this rule does not apply.
-  if (exclude_pattern_set_.MatchesURL(original_url))
+  if (exclude_pattern_set_.MatchesURL(original_url)) {
```

Comment:
this diff is not needed, existing formatting is acceptable

Reply:
I know but I just wanted to match everything up. It helps me a tiny bit when I want to add local logging to have these, and given that they're both equivalent I thought might as well. Personally I think always specifying {} is easier to read, but no strong feelings